site/src/examples/low-barrel/index.js

```diff
@@ -298,7 +302,7 @@
             });
 
         var multi = fc.series.multi()
-            .series([gridlines, line, area, brush])
```

Comment:
One thing to bear in mind is that a stroking an area will cause it to have an outline on all of its edges, whereas a combination of line & area gives the impression that only the 'top' edge of the area has been stroked. Although looking at the visuals I can't see that it makes any difference here.